Projekt/src/app/app.component.html

```diff
@@ -18,3 +18,13 @@ <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular bl
   </li>
 </ul>
 
+<h2>Here is your cards</h2>
+<div *ngIf="deck && pile">
+  <div *ngFor="let card of pile.cards">
+    <img src="{{card.image}}">
+  </div>
+</div>
+<div>
+  <input type="number" (input)="cardsNumber = $event.target.value" min="1" max="52">
```

Comment:
I think we could do something like this: `max="deck.remaining"`. Try if it works :)